### ISSUE TITLE
update demo code in readme to work with angularjs 1.8

### ISSUE THREAD

**Issue by ajtag** [ExLibrisGroup/primo-explore-devenv#143]
according to 
https://knowledge.exlibrisgroup.com/Primo/Product_Materials/Announcements/Preparing_for_the_Upgrade_to_Angular_1.8_in_Primo%2F%2FPrimo_VE

it looks like the demo code in the README will not be valid after the November2022 update

**Comment by rschwab** [comment]
Good catch! Yes, specifically the [code on this page](https://github.com/ExLibrisGroup/primo-explore-package/tree/master/VIEW_CODE/js) needs to use the onInit() method.
